scripts/generate_status_videos.py

```
@@ -0,0 +1,630 @@
+
+    if requested:
+        for key in requested:
+            if key not in keys_by_normalized:
+                keys_by_normalized[key] = {key}
+        keys = sorted(set(keys) | requested)
+
+    if args.limit > 0:
+        keys = keys[: args.limit]
+
+    if not keys:
+        raise RuntimeError("No status declarations found for the current selection.")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)